packages/cli/README.md

```diff
@@ -127,3 +127,86 @@ jobs:
       - run: npm ci
       - run: npx code-pushup autorun --upload.apiKey=${{ secrets.PORTAL_API_KEY }}
 ```
+
+# CLI
```

Comment:
There is already a heading level 1 (`#`) at the top of this document, so this should be level 2 (`##`), and the sub-sections should then be nested accordingly (`##` :arrow_right: `###`, etc.).

(We could integrate [markdownlint](https://github.com/DavidAnson/markdownlint) to these catch typograhic errors, BTW. I personally use the VSCode extension.)

Also, maybe "CLI reference" or "CLI commands and options" would help distinguish this section from what's above? :thinking:

Reply:
Agree. 

Create issue #320 about markdownlint